internal/service/imagebuilder/image_test.go

```diff
@@ -498,3 +521,130 @@ resource "aws_imagebuilder_image" "test" {
 }
 `, tagKey1, tagValue1, tagKey2, tagValue2))
 }
+
+func testAccImageContainerRecipeARNConfig(rName string) string {
+	return fmt.Sprintf(`
```

Comment:
I did not reuse the `testAccImageBaseConfig` as the base needed for container builds differs too much.